master-galera/master.cfg

```diff
@@ -391,15 +391,17 @@ for os_i in os_info:
     if "install_only" in os_info[os_i] and os_info[os_i]["install_only"]:
         continue
     for arch in os_info[os_i]["arch"]:
-        builder_name = "gal-" + arch + "-" + os_i
+        builder_name = "gal-" + arch + "-" + os_i.replace(".", "_")
 
         assert builder_name in builders_galera
 
-        worker_name = arch + "-bbw-docker-" + os_i
-        if arch == "amd64":
-            worker_name = "x64-bbw-docker-" + os_i
+        worker_name = "-bbw-docker-" + os_i.replace(".", "_")
```

Comment:
Please let's avoid hacking even more in BuildBot just to treat edge cases :(
It will be harder and harder to refactor the code and follow how names are resolved in the end.

I would discard (".", "_") related changes and keep it simple like:
- in os_info.yml --> opensuse-1505
- image tag in workflow --> 1505 also

Reply:
Ack, will force push back 7f9416a8f58922b01c0b5949018c319e66ec74bc once SUSE bug resolved.

Reply:
Thanks @grooverdan .
Now it came to my mind that this introduces a new s390x builder.
I don't know if we can afford one more given the large s390x queues, lets discuss this in the next internal meeting if we can do something infrastructure wise.

Until then, I'm sorry to postpone merging your patch. :(